bcipy/signal/model/inquiry_preview.py

```diff
@@ -2,30 +2,91 @@
+       The user's behavior is fully determined by their desired letter and the shown letters.
+       Only some edge effects such as frustration or fatigue are ignored by this.
+
+    We're interested in the alphabet posterior overall:
+
+        p(want | button, show, past) = p(button | want, show, past) p(want | show, past) / p(button | show, past)
+                                    # Use assumption 2.
+                                    = p(button | want, show) p(want | show, past) / p(button | show, past)
+                                    # Given past, show does not affect want
+                                    = p(button | want, show) p(want | past) / p(button | show, past)
+                                    # Will normalize over alphabet anyway. Can ignore normalization const.
+                                    \propto p(button | want, show) p(want | past)
+                                    # Leave out the prior - this is handled outside by evidence fusion.
+                                    \propto p(button | want, show)
+                                            ~~~~~~~~~~~~~~~~~~~~~~
+                                                    |
+                                    These are the multiplicative updates we'll return.
+
+
+    For shown letters, if the user gives a positive button response, we have terms like:
+
+        p(want=A | +, show=ABC, past) \propto p(+ | want=A, show=ABC)
+                                      \propto p(+ | want in show) = "probability of correct button press"
+
+    And for shown letters, a negative response gives us terms like this:
+
+        p(want=A | -, show=ABC, past) \propto p(- | want=A, show=ABC)
+                                      \propto p(- | want in show) = "probability of incorrect button press"
+
+    For unshown letter, positive button response:
+
+        p(want=Z | +, show=ABC, past) \propto p(+ | want not in show) = "probability of incorrect button press"
+
+    For unshown letter, negative button response:
+
+        p(want=Z | -, show=ABC, past) \propto p(- | want not in show) = "probability of correct button press"
+
+    Note that this simple rule, followed by normalizing the result vector, has the desired scaling property;
+    with larger inquiries, the prior probability of a "+" reponse is larger.
+    Thus, for a "+" response, the total probability given to the shown letters should grow as the inquiry length grows.
+
+    >>> def example(N, L):
+    ...     # Show N letters from an alphabet of length L
+    ...     probs = np.array([0.95]*N + [0.05] * (L-N))
+    ...     probs /= probs.sum()
+    ...     # Check the effect of a "+" response on the N shown letters
+    ...     res = probs[:N].sum()
+    ...     print(format(res, "0.2f"))
+    >>> example(2, 10)
+    0.83
+    >>> example(5, 10)
+    0.95
+    >>> example(8, 10)
+    0.99
 
     Args:
-        inquiry - the symbols presented in the inquiry preview
```

Comment:
These are glossary terms! This could be used for more than the alphabet as well. like a list of paths to images.

Reply:
https://github.com/CAMBI-tech/BciPy/tree/1.5.1#glossary

Reply:
Whoops - I had changed terminology while trying things out, because I had gotten confused - but I intended to revert that before starting PR. I'll change these back.